crawler.py

```diff
@@ -483,17 +590,19 @@ def parse_body(url, soup, response):
 
     return tag_indicators
 
+
 def parse_footer(url, soup, response):
     tag_indicators = []
 
-    footer = soup.find_all("footer")
+    footer = soup.find("footer")
```

Comment:
addresses the following error: 
`Exception occurred while running parse_footer: ResultSet object has no attribute 'get_text'. You're probably treating a list of elements like a single element. Did you call find_all() when you meant to call find()?`